src/lib/components/Search.svelte

```diff
@@ -0,0 +1,76 @@
+<script lang="ts">
+	export let search: string;
+	export let placeholder = 'Search...';
+	export let label: string | null = null;
+	export let name = `search_${Math.random().toString(36).substring(7)}`;
```

Comment:
This is valid for all subsequent input components.

We can discuss this with @clemcode, but I would personally rather have an `id` prop instead of `name`, and not have a random default value. If a required prop is forgotten, Svelte displays a warning in the console, and it also might display one if it finds two elements with the same id.

Reply:
I agree 👍